README.md

```diff
@@ -0,0 +1,213 @@
+# GIAPI OVERVIEW
+The GIAPI concept was developed by GEMINI to facilitate the interaction between the Telescope and the instrument software. Thanks to GIAPI, instrument developers do not need to know in detail the GEMINI Telescope Control System. They only need to deploy the GMP (Gemini Master Process) and incorporate GIAPI-GLUE in their source code to be able to communicate with the Telescope. 
+
+The GMP (Gemini Master Process) is a component provided by Gemini that provides most of the Gemini integration features and functionalities. It basically transforms messages sent through the GIAPI-GLUE to the Telescope system components. The language-specific glue is the API layer used by the builder to implement the integration functionality.  For more details of the GMP and GIAPI-GLUE it is recommended to read the documents "GIAPI Design and Use" and "GIAPI C++ Language Glue API". The below image depicts a Simple Instrument architecture to communicate with the Telescope.
+
+<p align="center">
+<img  align="center" src="https://raw.githubusercontent.com/gemini-hlsw/gmp/igrins2/images/SimpleGMPArhictecture.jpg">
+</p>
+
+# Desployment from binaries. 
+It is possible deploying the GMP and GIAPI-GLUEc++ from their binaries. Gemini recommend uses Rocky 8 or Centos 7 because these are the Operating Systems supported by GEMINI and for which the GIAPI-GLUEc++ binaries have been compiled. 
+
+If you want to deploy the system in other platform, you should compile the external libraries of the [GIAPI-GLUEc++](https://github.com/gemini-hlsw/giapi-glue-cc/tree/develop/external) for you platform. In the previous link is decribed the instrucction executed by GEMINI to CENTOS7 and ROCKY8.
+
+## Dependencies 
+The only dependencies we need for GMP and GIAPI-glue are as follows.
+* Java JDK 1.8
+* GCC greater than 4.8.1.
```

Comment:
Should it be a newer gcc that supports C++11?

Reply:
GCC 4.8.1 was the first feature-complete implementation of the 2011 C++ standard, previously known as C++0x. Therefore, the GIAPI-glue supports any GCC compiler greater than the 4.8.1 version.